src/app/_components/GoogleLogin/GoogleLoginButton.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { GoogleLoginButton } from "./GoogleLoginButton";
+import { Box } from "@mui/material";
+
+// More on how to set up stories at: https://storybook.js.org/docs/writing-stories#default-export
```

Comment:
Will need to remove the comments within the stories

Reply:
Comments are removed. Thanks